utils/promise-runner.js

```diff
@@ -76,19 +76,20 @@ function runOneSuccessful(listEntries, executeEntry, callback, debugInfo) {
 
   const entry = availableListEntries.shift();
 
-  const executeForSingleEntry = (entry) => {
-    return executeEntry(entry)
-      .then((result) => {
-        return callback(null, result);
```

Comment:
Calling the `callback` in the `then()` handler causes any errors thrown from the callback to be caught by the `catch()` handler which in turn will call the `callback` again, causing the "Callback called twice" exception